## Rule Details

This rule aims to standardise test description wording.

Examples of **incorrect** code for this rule:

```js

it('verify item added to basket');

```

Examples of **correct** code for this rule:

```js

it('should add item to basket');

```

## When Not To Use It

Not aware of any at the moment...
